src/services/clients/connector.client.js

```diff
@@ -420,6 +420,27 @@ ConnectorClient.prototype = {
     return response.data
   },
 
+  /**
+   * @param serviceExternalId {string}
+   * @param accountType {string}
+   * @param allowApplePay {boolean}
+   * @returns {Promise<Object>}
+   */
+  updateAllowApplePay: async function (serviceExternalId, accountType, allowApplePay) {
```

Comment:
There's a common `patchGatewayAccountByServiceExternalIdAndAccountType` method you could make use of, you would just need to update the `GatewayAccountUpdateRequest`class to have `allow_apple_pay`/`allow_google_pay` patches